src/lib/response-triggers/finish.ts

```diff
@@ -64,7 +71,6 @@ export default class Finish extends Trigger {
 		var reset = await this.reset.createMessage();
 
 		let message = `## Winners\n\n` + winnersMarkdown + '\n\n';
```

Comment:
Why not move the winners markdown up along with the results? You can have the final message be some kind of sanity check like "finished sending PMs"